Gemfile

```diff
@@ -26,6 +26,7 @@ group :development, :test do
   gem 'binding_of_caller'
   gem 'pry'
   gem 'capybara'
+  gem 'capybara-screenshot', :group => :test
```

Comment:
Sweet!! :sparkles: So excited about this capybara-screenshot! Never heard of it before seeing it here!